tasks/functions/functions.js

```diff
@@ -15,31 +15,44 @@ function sum(a, b) {
  * }
  */
 function getFullName(object) {
-
+	return object.firstName + " " + object.lastName;
 }
 
 /**
  * write fuction that checks if number is odd
  * true if odd, false if even
  */
 function isOdd(n) {
-
+	return (n % 2 == 1) ? true : false;
 }
 
 /**
  * write function that return shortest of words in the given array
  * e.g ["one", "two", "three"] should return one
  */
 function getShortest(wordArray) {
-
+	let counter = wordArray[0].length;
+	let shorter = wordArray[0];
+	for (let i = 1; i <  wordArray.length; i++) {
+	  if (wordArray[i].length < counter){
+		counter = wordArray[i].length;
+		shorter = wordArray[i];
+	  }
+	} 
+	return shorter;
 }
 
 /**
  * write function that returns word google with given numbers of "o" symbols
  * e.g getGoogle(5) should return "gooooogle"
  */
 function getGoogle(n) {
-
+	let google = "g";
+	for (let i = 0; i < n; i++){
+	  google += "o";
+	}
+	google += "gle"
+	return google;
 }
```

Comment:
JFYI

By using `template strings` and `repeat` method:

`function getGoogle(n) {
	return `g${"o".repeat(n)}gle`
}
`